code/game/dna/dna_modifier.dm

```diff
@@ -271,15 +271,16 @@
 
 		occupant.notify_ghost_cloning(source = src)
 
-/obj/machinery/dna_scannernew/proc/go_out()
+/obj/machinery/dna_scannernew/proc/go_out(force)
```

Comment:
pass the user arg so we're not using usr

Reply:
Updated. Please verify that it works the way you want.